clipbit.py

```diff
@@ -23,13 +27,34 @@
     default=False,
     help="Verbose mode: shows progress bar and other information.",
 )
-def generate_summary(summarize, verbose):
-    captions_obj = Captions(summarize)
-
-    # TO-DO
-    # captions_obj.video_path + "/script.txt" is the path to the combined script.
-    # import your file and then
-    # call your NLP model function here.
+@click.option(
+    "-d/-c",
+    "--detailed/--clean",
+    default=False,
+    help="Detailed mode: displays title, captions and summary in a table.",
+)
+def generate_summary(summarize, verbose, detailed):
+    verboseprint = print if verbose else lambda *a, **k: None
+
+    captions = Captions(summarize, verbose)
+    if captions.srt_filename:
+        final_summary = ts(captions.video_path, verbose)
+        if final_summary:
+            if detailed:
+                tabulate_summary(final_summary, captions)
+            else:
+                print(f"\n{final_summary}\n")
+            verboseprint(
+                "\033[96m\u2192 Captions, Script and Summary are all available under videos/<video_name>/ directory.\033[0m"
+            )
+
+
+def tabulate_summary(summary, captions):
```

Comment:
What does this function do? Remember to insert docstrings. This allows the reviewer to have a good understanding of what your function attempts to achieve